test/commands/graph_spec.rb

```diff
@@ -273,7 +273,7 @@
 
       it "matches tag case-insensitively and scales the graph" do
         stdout_only <<-OUTPUT
-Nov 2015 |█████∙∙∙∙∙∙∙∙∙∙∙∙∙∙∙|
+Nov 2015 |∙∙∙∙∙∙∙∙∙∙∙∙∙∙∙∙∙∙∙∙|
```

Comment:
I don't think I understand why this needed to change.

Reply:
This was something I forgot to explain. And it could be a case of me being too precious about the test descriptions. 

When I removed `Marie's Diner` in favor of `Martha's Vineyard`, the line:
`**Grace Hopper** and I went to _LOCATION_. George had to cancel at the last minute` no longer had anything to do with food, and so I removed the `@food` tag from the line. The tests then had to change to reflect this change.

Reply:
Makes sense to me; thanks!